package.json

```diff
@@ -20,16 +20,16 @@
   },
```

Comment:
We should maybe add the tslint checks to the CI tests so no code that doesn't pass the linter gets pushed ?

Reply:
@Slamper done 👍 thanks for your advice.